Index: src/main/java/com/team/GroundTruth/entity/HazardReport/HazardReport.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/team/GroundTruth/entity/HazardReport/HazardReport.java b/src/main/java/com/team/GroundTruth/entity/HazardReport/HazardReport.java
new file mode 100644
--- /dev/null	(date 1769889394365)
+++ b/src/main/java/com/team/GroundTruth/entity/HazardReport/HazardReport.java	(date 1769889394365)
@@ -0,0 +1,15 @@
+package com.team.GroundTruth.entity.HazardReport;
+
+import jakarta.persistence.*;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "hazard_reports")
+public class HazardReport {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @ManyToOne
+}
Index: src/main/java/com/team/GroundTruth/entity/User/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/team/GroundTruth/entity/User/User.java b/src/main/java/com/team/GroundTruth/entity/User/User.java
new file mode 100644
--- /dev/null	(date 1769889180823)
+++ b/src/main/java/com/team/GroundTruth/entity/User/User.java	(date 1769889180823)
@@ -0,0 +1,24 @@
+package com.team.GroundTruth.entity.User;
+
+
+import jakarta.persistence.*;
+
+import java.util.ArrayList;
+import java.util.UUID;
+
+@Entity
+@Table(name="users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @Column(nullable = false, name="username", unique=true, length = 50)
+    private String username;
+
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<HazardReport> reports = new ArrayList<>();
+
+
+}
Index: src/main/java/com/team/GroundTruth/repository/Repository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/team/GroundTruth/repository/Repository.java b/src/main/java/com/team/GroundTruth/repository/Repository.java
new file mode 100644
--- /dev/null	(date 1769888449803)
+++ b/src/main/java/com/team/GroundTruth/repository/Repository.java	(date 1769888449803)
@@ -0,0 +1,7 @@
+package com.team.GroundTruth.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface Repository extends JpaRepository {
+
+}
Index: src/main/java/com/team/GroundTruth/entity/Hazard/Hazard.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/team/GroundTruth/entity/Hazard/Hazard.java b/src/main/java/com/team/GroundTruth/entity/Hazard/Hazard.java
new file mode 100644
--- /dev/null	(date 1769888403837)
+++ b/src/main/java/com/team/GroundTruth/entity/Hazard/Hazard.java	(date 1769888403837)
@@ -0,0 +1,15 @@
+package com.team.GroundTruth.entity.Hazard;
+
+
+import jakarta.persistence.*;
+
+import java.util.UUID;
+
+@Entity
+@Table(name="Hazards")
+public class Hazard {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+}
Index: src/main/java/com/team/GroundTruth/entity/Hazard/Type.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/team/GroundTruth/entity/Hazard/Type.java b/src/main/java/com/team/GroundTruth/entity/Hazard/Type.java
new file mode 100644
--- /dev/null	(date 1769888334595)
+++ b/src/main/java/com/team/GroundTruth/entity/Hazard/Type.java	(date 1769888334595)
@@ -0,0 +1,4 @@
+package com.team.GroundTruth.entity.Hazard;
+
+public enum Type {
+}
Index: src/main/java/com/team/GroundTruth/entity/Hazard/Severity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/team/GroundTruth/entity/Hazard/Severity.java b/src/main/java/com/team/GroundTruth/entity/Hazard/Severity.java
new file mode 100644
--- /dev/null	(date 1769888348540)
+++ b/src/main/java/com/team/GroundTruth/entity/Hazard/Severity.java	(date 1769888348540)
@@ -0,0 +1,4 @@
+package com.team.GroundTruth.entity.Hazard;
+
+public enum Severity {
+}
